### ISSUE TITLE
[REF-3190] Add testing to verify props propagation of all reflex components

### ISSUE THREAD

**Issue by Lendemor** [reflex-dev/reflex#3564]
Creating this issue to track an idea suggested in #3541.

<sub>[REF-3190](https://linear.app/reflex-dev/issue/REF-3190/add-testing-to-verify-props-propagation-of-all-reflex-components)</sub>

**Comment by benedikt-bartscher** [comment]
@abulvenz in https://github.com/reflex-dev/reflex/issues/3541 you mentioned "In the first draft I tried to do it with automatic component instanciation, but as I wrote there are many exceptions from the rule.".

Maybe we can introduce a `create_fake`/`create_demo` classmethod in `rx.Component` which raises `NotImplementedError` by default. Every component which has a `create_fake`/`create_demo` method could then be included in a dynamic pytest fixture. In the future, we could also use this classmethod to generate docs/examples.

**Comment by abulvenz** [comment]
> @abulvenz do you want me to assign this to you? (since you have a draft working on it)

Thanks for asking @Lendemor . Right now this has low prio to me. If you assign it to me, it might get done in some bright future when I find the time.

**Comment by ShubhamAXS19** [comment]
can i work on this issue ??

**Comment by picklelo** [comment]
@ShubhamAXS19 yes, just assigned you! Let us know if you have issues / questions

**Comment by picklelo** [comment]
@ss-mathew just assigned you!

**Comment by masenf** [comment]
complete prop testing will be added for the new component library, we won't be backfilling coverage for radix or older components. their behavior is currently pinned by a handful of integration tests.
